bedrock/firefox/templates/firefox/browsers/quantum.html

```diff
@@ -0,0 +1,55 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/browsers/quantum" %}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ ftl('page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('page-desc') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('quantum') }}
+{% endblock %}
+
+{% block content %}
+
+{% call hero(
+  title=ftl('the-latest-firefox'),
+  class='mzp-has-image',
+  image_url='img/firefox/new/desktop/ie9/browser.jpg',
+  include_highres_image=True,
+  include_cta=True,
+  heading_level=1
+) %}
+<p>{{ ftl('firefox-quantum-was') }}</p>
+{{ download_firefox() }}
```

Comment:
Suggest using the simple download button here: https://bedrock.readthedocs.io/en/latest/download-buttons.html